src/AdminAssets.php

```diff
@@ -39,7 +39,7 @@ private static function enqueue_menu_editor_script(): void {
 						'maxChars' => 18,
 					],
 					1 => [
-						'maxItems' => 5,
+						'maxItems' => 10,
```

Comment:
As it's been requested by the ticket, the maximum items should be set to `10` but scrollable greater than `6`.